tgui/packages/tgui/interfaces/AresInterface.js

```diff
@@ -0,0 +1,679 @@
+import { useBackend } from '../backend';
+import { Flex, Box, Section, Button } from '../components';
+import { Window } from '../layouts';
+
+const PAGES = {
+  'login': () => Login,
+  'main': () => MainMenu,
+  'announcements': () => AnnouncementLogs,
+  'bioscans': () => BioscanLogs,
+  'bombardments': () => BombardmentLogs,
+  'apollo': () => ApolloLog,
+  'access_log': () => AccessLogs,
+  'delete_log': () => DeletionLogs,
+};
+
+export const AresInterface = (props, context) => {
+  const { data } = useBackend(context);
+  const { current_menu } = data;
+  const PageComponent = PAGES[current_menu]();
+
+  return (
+    <Window theme="weyland" width={780} height={725}>
```

Comment:
consider using a crt theme like `crtgreen` or `crtblue` since they give off a more terminal vibe than the weyland theme.